lib/ig.js

```diff
@@ -77,7 +77,21 @@ function exportIG(specifications, fhirResults, outDir, configuration = {}, specP
       location: 'http://hl7.org/fhir/us/core',
       version: '1.0.1'
     }];
+    if (config.implementationGuide.dependencies != null) {
```

Comment:
I assumed it would not work because of the argo comments. When I tried it, using argo still did not work, so I left that part commented out. However, it seemed this was a problem specific to argo, not generally with loading in an external dependency, so I was able to factor the block of code out of the `if`s here https://github.com/standardhealth/shr-fhir-export/pull/166/commits/fb40f694fe3e123d87ba5062a0052b55367a86f3.